pyscitt/pyscitt/cli/submit_signed_claims.py

```diff
@@ -14,27 +14,39 @@ def submit_signed_claimset(
     client: Client,
     path: Path,
     receipt_path: Optional[Path],
+    receipt_type: str,
     service_trust_store_path: Optional[Path],
     skip_confirmation: bool,
 ):
     if path.suffix != ".cose":
-        raise ValueError("unsupported file extension")
+        raise ValueError("unsupported file extension, must end with .cose")
+
+    if receipt_type == "raw":
```

Comment:
And then here use the enum values instead of hardcoded strings:

```suggestion
    if receipt_type == ReceiptType.RAW.value:
```

Reply:
done